### PR TITLE
F/vagrant box

### PR DESCRIPTION
**Pull request by BenoitRichard1212**
Added the setup for the Vagrant box.

Readme.md included with installation instruction, future features and contact info.

### CODE REVIEW

**Comment by fariasf** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/fariasf/drdt-test/325/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Lighthouse test passed! The score of `92` isn't less than the acceptable score of `89` (`5` less than the score of `94` on the master branch)

View the full [Lighthouse performance report for `pull/6`](https://circleci.com/gh/fariasf/drdt-test/326/artifacts/0/artifacts/lighthouse_results/pr-6/lighthouse.report.html) and compare it to the [Lighthouse performance report for the `master` branch](https://circleci.com/gh/fariasf/drdt-test/326/artifacts/0/artifacts/lighthouse_results/master/lighthouse.report.html)

File: drdt-vagrant/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	UN : local.admin
+	PW : password
+
+	** PLEASE NOTE THAT CURRENTLY THE DRDT CODE IS PULLED FROM THE MASTER GIT BRANCH **
+
+
+Upcomming features 
+___________________
+
+	Different env. setup : (Test, Staging, Prod).
```
Comment (fariasf) [inline drdt-vagrant/README.md:79]: I think this is already taken care (for the implemented features) via environment variables!

**Comment by fariasf** [comment]
So far LGTM! Going to test it now. It seems it includes the full WordPress source code. Is that intentional? Why not using composer?

File: drdt-vagrant/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	5 ) You will need a valid copy of the CPT database you can download one here :
+
+		http://staging.constructionprotips.com/wp-content/uploads/sites/9/2018/12/CPT_dump.sql
+
+		*Currently some issue downloading the file, I'm working on finding a good place to store this. Will probably be in one of our s3 storage on AWS.
+
+		you will need to place that file in the drdt-vagrant/ folder
+	
+	6 ) You will need to clone the drdt-test project inside the drdt-vagrant directory.
```
Comment (fariasf) [inline drdt-vagrant/README.md]: The VM is in the same repo as the code. Mind clarifying?

**Comment by fariasf** [comment]
LGTM!! Tested it and works like a charm. Minor request: can we have the Vagrantfile in the root project directory? Also, please see comment about including full WP source code. @BenoitRichard1212

File: drdt-vagrant/README.md (hunk 2 above)
Comment (BenoitRichard1212) [reply drdt-vagrant/README.md]: This is the way I have always seen a vagrant config, dvlpr-vagrant works the same.
I'll look into getting this to run one level above!

File: drdt-vagrant/README.md (hunk 1 above)
Comment (BenoitRichard1212) [reply drdt-vagrant/README.md:79]: Yes, I think only small hotfixes may be needed to make sure everything is as we wish.

**Comment by fariasf** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/fariasf/drdt-test/333/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Lighthouse test passed! The score of `94` isn't less than the acceptable score of `89` (`5` less than the score of `94` on the master branch)

View the full [Lighthouse performance report for `pull/6`](https://circleci.com/gh/fariasf/drdt-test/332/artifacts/0/artifacts/lighthouse_results/pr-6/lighthouse.report.html) and compare it to the [Lighthouse performance report for the `master` branch](https://circleci.com/gh/fariasf/drdt-test/332/artifacts/0/artifacts/lighthouse_results/master/lighthouse.report.html)

**Pull request merged**